test/t/feeds/feeds_live_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule T.FeedsLiveTest do
-  use T.DataCase, async: true
```

Comment:
This tests interfered with `feed_channel_live_test.exs` so I made it run separately.